features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/configureroom/ConfigureRoomNode.kt

```diff
@@ -58,11 +60,13 @@ class ConfigureRoomNode @AssistedInject constructor(
     @Composable
     override fun View(modifier: Modifier) {
         val state = presenter.present()
+        val context = LocalContext.current
         ConfigureRoomView(
             state = state,
             modifier = modifier,
             onBackPressed = this::navigateUp,
             onRoomCreated = this::onRoomCreated,
+            onOpenSystemSettings = { context.openAppSettingsPage() }
```

Comment:
Maybe this can be moved into PermissionsPresenter itself?

Reply:
yes, a bit simpler: dd5d67d18622d00c87ad4a25b00e9bbde97a7f93